reconcile/openshift_resources_base.py

```diff
@@ -44,7 +67,10 @@
 from reconcile.utils.secret_reader import SecretReader
 from reconcile.utils.semver_helper import make_semver
 from reconcile.utils.sharding import is_in_shard
-from reconcile.utils.vault import SecretVersionIsNone, SecretVersionNotFound
+from reconcile.utils.vault import (
+    SecretVersionIsNone,
+    SecretVersionNotFound,
+)
```

Comment:
why this lint changes? where do they come from?

Reply:
I think in my previous PR I had ruff setup incorrectly with the vscode extension. When I ran `make format` this time, it outputted these changes

Reply:
weird, I run `make format` locally and I don't get any change...

Reply:
User error on my part :rofl:

Reply:
it's fine, it got me a bit confused. But the new syntax is the most widely used in the code.